client/pages/user-reviews.jsx

```diff
@@ -40,7 +44,41 @@ export default class UserReviews extends React.Component {
 
   handleClose(event) {
     this.setState({
-      show: false
+      show: false,
+      showDelete: false
+    });
+  }
+
+  handleDelete(event) {
+    const parkCode = this.state.deleteId;
+    const token = window.localStorage.getItem('park-reviews-jwt');
+    const request = {
+      method: 'delete',
+      headers: {
+        'X-Access-Token': token
+      }
+    };
+    fetch(`/api/reviews/${parkCode}`, request)
+      .then(response => response.json())
+      .then(result => {
+        this.retrieveReviews();
+        this.setState({
+          showDelete: false
+        });
+      })
```

Comment:
Rather that re-fetching the reviews list, it would be more efficient to filter the one you already have and update your state.